lumen/ai/views.py

```diff
@@ -88,6 +88,9 @@ async def _render_component(self):
             return
 
         try:
+            if self.component is None or self.spec is None:
```

Comment:
Seems like `self.spec is None` is already handled above, seems like what actually needs to happen here is for the `self._rendered` check below to be changed.